src/main/resources/application.properties

```diff
@@ -4,8 +4,8 @@
 #local
 spring.datasource.url=jdbc:h2:mem:testdb
 spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=postgres
-spring.datasource.password=ENC(Q95IwHemI5qChQtfslDvug==)
+#spring.datasource.username=postgres
+#spring.datasource.password=ENC(Q95IwHemI5qChQtfslDvug==)
```

Comment:
To chyba niepotrzebnie zakomentowane xd

Reply:
Wywala mi to jak jest odkomentowane, ale mogę przywrócić tak jak wcześniej było